src/triggers/TDTM_Batch.trigger

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
The Batch__c sObject, I believe, is mostly-entirely deprecated at this point as it serves the now sunsetted Batch Data Entry feature. Maybe we cover with a stub trigger anyway but just wanted to make sure we were clear that the Batch__c object is no longer really in service.